README.md

```diff
@@ -328,141 +331,151 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
+                    Setpoint item=LandroidScheduleThursdayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleThursdayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleThursdayEdgecut
+                }
+                Frame label="Schedule Friday" {
+                    Setpoint item=LandroidScheduleFridayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleFridayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleFridayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleFridayEdgecut
+                }
+                Frame label="Schedule Saturday" {
+                    Setpoint item=LandroidScheduleSaturdayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleSaturdayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleSaturdayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleSaturdayEdgecut
+                }
+                Frame label="Schedule Sunday" {
+                    Setpoint item=LandroidScheduleSundayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleSundayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleSundayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleSundayEdgecut
+                }
             }
+            Slider item=LandroidRainDelay minValue=0 maxValue=750 step=30
+        }
+        Frame label="Statistic" {
+            Text item=LandroidTotalTime
+            Text item=LandroidTotalDistance label="Total Distance [%.2f km]"
+            Text item=LandroidTotalBladeTime
         }
-        Setpoint item=LandroidRainDelay minValue=0 maxValue=750 step=30
-    }
-    Frame label="Statistic" {
-        Text item=LandroidTotalTime
-        Text item=LandroidTotalDistance label="Total Distance [%.2f km]"
-        Text item=LandroidTotalBladeTime
-    }
-    Frame label="Orientation" {
-        Text item=LandroidPitch
-        Text item=LandroidRoll
-        Text item=LandroidYaw
     }
 }
 ```
 
 ### .rules
+
+Update Landroid Status to reflect in main menu
+
+<img src="images/SC_BasicUI_MainStatus.png" width="50%">
+<br>
+
 ```
-TODO
+rule MowerStatus
+when
+	Item LandroidErrorCode changed or 
+	Item LandroidStatusCode changed
+then
+	if (LandroidErrorCode.state != 0) {
+		Shaun.postUpdate(transform("MAP", "landroid_error_de.map", LandroidErrorCode.state.toString))
+	} else {
+		Shaun.postUpdate(transform("MAP", "landroid_status_de.map", LandroidStatusCode.state.toString))
+	}
+end
 ```
 
+Place the following *.map to your ..\conf\transform
+
+1. [landroid_error_de.map](https://github.com/nibi79/worxlandroid/doc/landroid_error_de.map)
```

Comment:
These files are not in your pull request

Reply:
Hmm, they should but not in images but in docs. Will check this

Reply:
They are in my rep under docs, probabaly I don't have rights to upload those. As these are not images, how about a new folder "files" for this? Or where shall I place them?

Reply:
This would leave "images" for Screenshots and "files" for all which is required to use this configuration.